### PR TITLE
Core 690 twin creation dispatcher

### PR DESCRIPTION
**Pull request by karladler**

### CODE REVIEW

File: dapps/assets/src/components/digitaltwins/AddDigitalTwin.ts [hunk 1]
```diff
@@ -116,24 +75,30 @@ class AddDigitalTwinComponent extends mixins(EvanComponent) {
...
+      return;
     }
 
-    this.selectedTemplate = null;
+    this._setDefaults();
+    this._addToPresetTemplates(this.template, files[0].name);
+    this.selectedTemplate = files[0].name;
+
+    // ugly hack to wait till underlying select component received new props
+    window.setTimeout(() => this.$forceUpdate(), 200);
```
Comment (Tschuck) [inline dapps/assets/src/components/digitaltwins/AddDigitalTwin.ts]: Ist this also working?

```suggestion
    this.$nextTick(() => this.$forceUpdate(), 200);
```

File: dapps/assets/src/components/dispatchers/createTwinDispatcher.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  Fifth Floor, Boston, MA, 02110-1301 USA, or download the license from
+  the following URL: https://evan.network/license/
+*/
+
+import { DigitalTwin, DigitalTwinOptions } from '@evan.network/api-blockchain-core';
+import * as dappBrowser from '@evan.network/ui-dapp-browser';
+import { Dispatcher, DispatcherInstance } from '@evan.network/ui';
+
+const dispatcher = new Dispatcher(
+  `assets.${ dappBrowser.getDomainName() }`,  // TODO: verify if its correct
```
Comment (Tschuck) [inline dapps/assets/src/components/dispatchers/createTwinDispatcher.ts]: For now it's correct :)
Maybe we want to move it later to the new twin dapp.
```suggestion
  `assets.${ dappBrowser.getDomainName() }`,
```

File: dapps/assets/src/components/dispatchers/createTwinDispatcher.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    }
+  })
+  // create the twin
+  .step(async (instance: DispatcherInstance, { twinTemplate }) => {
+    const newTwin = await DigitalTwin.create(instance.runtime as DigitalTwinOptions, {
+      accountId: instance.runtime.activeAccount,
+      containerConfig: { accountId: instance.runtime.activeAccount },
+      ...twinTemplate
+    });
+  });
```
Comment (Tschuck) [inline dapps/assets/src/components/dispatchers/createTwinDispatcher.ts]: Can we mark the new twin as favorite?

```suggestion
      await newTwin.addAsFavorite();
    });
```

**Comment by Tschuck** [comment]
Should we show a informatione, that currently a twin gets created and should we open this new twin directly after creation?

Possibilities:
- watch for finishing dispatcher in the add component and simple redirect to new twin
- add list entry for twins that are in creation to assts overview
- show modal for asking to open the twin

If i'ts too much, we can handle this one in a separated pull request.

File: dapps/assets/src/components/dispatchers/createTwinDispatcher.ts (hunk 3 above)
Comment (karladler) [reply dapps/assets/src/components/dispatchers/createTwinDispatcher.ts]: sure, but do we still need it with our chain indexing service?

**Comment by karladler** [comment]
> Should we show a informatione, that currently a twin gets created and should we open this new twin directly after creation?
> 
> Possibilities:
> 
> * watch for finishing dispatcher in the add component and simple redirect to new twin
> * add list entry for twins that are in creation to assts overview
> * show modal for asking to open the twin
> 
> If i'ts too much, we can handle this one in a separated pull request.

1 and 3 for sure, also the loading state and reset of the formular needs to be implemented. 
For adding the twin to the assets I would try to perform the request to elastic as soon as the twin has been created. But new PR for this would be probably better.

File: dapps/assets/src/components/digitaltwins/AddDigitalTwin.ts (hunk 1 above)
Comment (karladler) [reply dapps/assets/src/components/digitaltwins/AddDigitalTwin.ts]: nope, but resolved with way better nextTicket on select component, thanks for the hint! 🎉

File: dapps/assets/src/components/dispatchers/createTwinDispatcher.ts (hunk 3 above)
Comment (karladler) [reply dapps/assets/src/components/dispatchers/createTwinDispatcher.ts]: other opinions here @DanielMaass @S3bb1  ?

Comment (Tschuck) [reply dapps/assets/src/components/dispatchers/createTwinDispatcher.ts]: Very good question, i forgot this totally.
Will be displayed at the top of the search list automatically, so a favorite would not be nessecary.

**Pull request merged**